Userland/Applications/PixelPaint/Tools/Tool.cpp

```diff
@@ -34,19 +34,19 @@ void Tool::on_keydown(GUI::KeyEvent& event)
     switch (event.key()) {
     case KeyCode::Key_LeftBracket:
         if (m_primary_slider)
-            m_primary_slider->set_value(m_primary_slider->value() - 1);
+            m_primary_slider->decrease_slider_by(1);
         break;
     case KeyCode::Key_RightBracket:
         if (m_primary_slider)
-            m_primary_slider->set_value(m_primary_slider->value() + 1);
+            m_primary_slider->increase_slider_by(1);
         break;
     case KeyCode::Key_LeftBrace:
         if (m_secondary_slider)
-            m_secondary_slider->set_value(m_secondary_slider->value() - 1);
+            m_primary_slider->decrease_slider_by(1);
```

Comment:
You probably want to set the secondary slider instead of the primary slider.

Reply:
Thanks!! I'll fix it (: